pico_w/wifi/blink/CMakeLists.txt

```diff
@@ -31,3 +31,24 @@ pico_add_extra_outputs(picow_blink_slow_clock)
 
 # add url via pico_set_program_url
 example_auto_set_url(picow_blink_slow_clock)
+
+# This version should behave exactly the same, but it runs the sys clock slowly.
+add_executable(picow_blink_fast_clock
+        picow_blink_fast_clock.c
+        )
+target_link_libraries(picow_blink_fast_clock
+        pico_stdlib              # for core functionality
+        pico_cyw43_arch_none     # we need Wifi to access the GPIO, but we don't need anything else
+        hardware_clocks
+        )
+# This requires us to modify the pio divisor to successfully communicate with the cyw43 chip
+target_compile_definitions(picow_blink_fast_clock PRIVATE
+        CYW43_PIO_CLOCK_DIV_INT=4
+        CYW43_PIO_CLOCK_DIV_FRAC=0
```

Comment:
probably use FRAC8 or whatever the correct new define is

Reply:
Yeah, a `git grep` says that `CYW43_PIO_CLOCK_DIV_FRAC8` is correct:
```
src/rp2_common/pico_cyw43_driver/include/pico/cyw43_driver.h:// PICO_CONFIG: CYW43_PIO_CLOCK_DIV_FRAC8, Fractional part of the clock divider for communication with the wireless chip 0-255, type=int, min=0, max=255, default=0, group=pico_cyw43_driver
```